numismatic/events.py

```diff
@@ -28,10 +30,24 @@ class Heartbeat(Event):
 
 
 @attr.s(slots=True)
-class Trade(Event):
+class PriceUpdate(Event):
```

Comment:
I would like to standardise on all the events. In my document, I have them all there. I see that once we get the wiki up, the community can propose new events as well

Reply:
👍